runtime/src/main/java/org/corfudb/runtime/view/OrderedGUIDGenerator.java

```diff
@@ -0,0 +1,21 @@
+package org.corfudb.runtime.view;
+
+import java.util.UUID;
+
+/**
+ * Globally Unique Identity (GUID) generator that returns GUIDs having a notion of
+ * a comparable global ordering.
+ *
+ * Created by Sundar Sridharan on 5/22/19.
+ */
+public interface OrderedGUIDGenerator {
```

Comment:
Please rename to `OrderedGuidGenerator` in the other case checkstyle will show you a warning

Reply:
Done. Sorry I missed that from the previous round!